pkg/spanconfig/spanconfigsqltranslator/sqltranslator.go

```diff
@@ -98,6 +98,16 @@ func (s *SQLTranslator) Translate(
 	return entries, translateTxn.CommitTimestamp(), nil
 }
 
+// descLookupFlags is the set of look up flags used when fetching descriptors.
```

Comment:
I'm not sure that this is true at all. It's not true for mutable descriptor lookups (they simply don't bother with leasing at all and for good reason) and off the top of my head I don't think it's true for immutable lookups either (we don't always want dropped descriptors, do we?).

I would change this description, and also rename `descLookupFlags`. If that's too much trouble, it's fine to get rid of this common definition completely.

Reply:
> we don't always want dropped descriptors, do we?

This is just for this `spanconfigsqltranslator` where I think we do always want dropped tables.

Reply:
Ahh never mind then, I thought this was in the `descs` package. Thanks for pointing that out.